labshare-client/src/App.vue

```diff
@@ -10,7 +10,7 @@
     "signOut": "Logout",
     "dataProtection": "Data protection",
     "imprint": "Imprint",
-    "request": "Request Ressources",
+    "requestRessource": "Request Ressources",
```

Comment:
It's spelled "resource" in English. We're also using English keys, so it should be corrected in bot hplaces